test/issue1158-stdin-for-single-files/stdin.d

```diff
@@ -0,0 +1,7 @@
+/+ dub.sdl:
+	name "hello"
```

Comment:
The name is optional since 1.6

Reply:
By removing name attribute, the file not longer compiles. I tried different formats:
```
/+ dub.sdl:
+/
```

```
/+ dub.sdl
+/
```

Reply:
Ah sorry. Apparently one items is needed then :/